functions/src/lobby.ts

```diff
@@ -439,20 +440,41 @@ export const onVoteWrite = functions.firestore.document("/lobbies/{code}/votes/{
       return;
     }
 
-    const { players, uids } = lobbyData;
+    const { players, uids, skipVote: oldSkipVote } = lobbyData;
+    let skipVote = oldSkipVote;
```

Comment:
why create an alias if you create another variable with the same name?

Reply:
I think this is done so that it is mutable but maybe it would be better to do a separate deconstruction with let